backend/services/user/rider.js

```diff
@@ -28,7 +39,7 @@ const riderCreate = async (request, response) => {
 
 const riderLogin = async (request, response) => {
```

Comment:
Rider login needs to return part-time/full-time status as well. Maybe something like that
`
        SELECT rider_id, 'rider' AS type, 'part' AS status
        FROM users JOIN riders ON user_id = rider_id NATURAL JOIN part_time_riders
        WHERE email = $1
        AND password = $2
        UNION
        SELECT rider_id, 'rider' AS type, 'full' AS status
        FROM users JOIN riders ON user_id = rider_id NATURAL JOIN full_time_riders
        WHERE email = $1
        AND password = $2
`